.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: rust
+cache: cargo
```

Comment:
Thought I'd add this... unless you deliberately have it off it should speed up builds (particularly since I'm adding even more stuff)